demos/demo-react-ts/src/hooks/useCti.ts

```diff
@@ -29,6 +29,9 @@ export type CompanyIdMatch = {
   name: string;
 };
 
+const INCOMING_NUMBER_KEY = "__react_demo_app_incoming_number";
+const INCOMING_CONTACT_NAME_KEY = "__react_demo_app_incoming_contact_name";
```

Comment:
Should we follow the naming convention for local storage keys here? This could be `LocalSettings:Calling:DemoReact:incomingNumber` and `LocalSettings:Calling:DemoReact:incomingContactName`.

Reply:
i can do it.

Reply:
Done